sdk/identity/identity/src/credentials/managedIdentityCredential/fabricMsi.ts

```diff
@@ -50,7 +50,7 @@ function prepareRequestOptions(
     method: "GET",
     headers: createHttpHeaders({
       Accept: "application/json",
-      Secret: process.env.IDENTITY_HEADER
+      secret: process.env.IDENTITY_HEADER
```

Comment:
This is fine. .NET uses this header in lowercase: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/identity/Azure.Identity/src/ServiceFabricManagedIdentitySource.cs#L74